src/global.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.scss' {
```

Comment:
@alokesh985 Can you explain this?

Reply:
![image](https://user-images.githubusercontent.com/56512633/130558594-773200fa-94a2-407e-8d41-1d2f2801f624.png)

I was having some difficult while importing .scss files into Typescript. So, I looked it up online. I found this.
However, now, it seems to be working even after removing this file.